deploy/manifests/support-bundle-manager.yaml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: support-bundle-sample
+  namespace: harvester-system
+  labels:
+    app: support-bundle-manager
+    harvesterhci.io/supportbundle: sample
+spec:
+  selector:
+    app: support-bundle-manager
+    harvesterhci.io/supportbundle: sample
+  ports:
+    - protocol: TCP
+      port: 8080
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: support-bundle-manager
+    harvesterhci.io/supportbundle: sample
+  name: supportbundle-manager-sample
+  namespace: harvester-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: support-bundle-manager
+  template:
+    metadata:
+      labels:
+        app: support-bundle-manager
+        harvesterhci.io/supportbundle: sample
+    spec:
+      containers:
+      - args:
+        - /usr/bin/support-bundle-utils
+        - manager
+        - --standalone
+        env:
+        - name: HARVESTER_NAMESPACE
+          value: harvester-system
+        - name: HARVESTER_VERSION
+          value: unknown (from manifest)
```

Comment:
Can we get the version from harvester setting?

Reply:
Thanks for the nice tip. I'll get the version from the `server-version` setting and ignore any error in case the backend has some trouble.

Reply:
Ah, it's not supposed to see a backend error because it's a CR and I can get it from Kubernetes. Thanks again.

Reply:
Fixed in https://github.com/harvester/support-bundle-utils/pull/6/commits/d842a6a2da31dc7f20353c194b169f0c78184c2e